src/backend/distributed/shared_library_init.c

```diff
@@ -188,6 +196,32 @@ _PG_init(void)
 
 
 /*
+ * multi_log_hook intercepts postgres log commands. We use this to show the user a
+ * meaningful error message when a process receives a SIGINT from the distributed
+ * deadlock detector.
```

Comment:
I think this is too specific. Maybe something like "We use this to override postgres error messages when they're not specific enough"? And then in the *body* describe the specific override? Because I'd be surprised if there weren't future ones coming...